inspect.cpp

```diff
@@ -665,6 +665,17 @@ namespace Sass {
     append_token("null", n);
   }
 
+  void Inspect::operator()(Parent_Selector* p)
+  {
+    if (p->selector()) {
+      p->selector()->perform(this);
+      append_string(";");
```

Comment:
Should use `append_delimiter` since it will add optional space in certain output modes!